nixos/modules/programs/nano.nix

```diff
@@ -2,6 +2,7 @@
 
 let
   cfg = config.programs.nano;
+  LF = "\n"
```

Comment:
A `;` is missing here.

Reply:
`nox-review` shouldn't be needed here: it's only needed if you modify a package expression as it rebuilds all packages depending on the altered package.

Reply:
Got it. I was just looking for a way to test my changes (for syntax errors at least :laughing:).

Reply:
I usually do this when testing module changes:

```
{
  test-vm = { pkgs, lib, config, ... }: {
    programs.nano.syntaxHighlight = true;
  };
}
```

And then run something like this:

```
NIX_PATH=nixpkgs=/path/to/nixpkgs/checkout nixos-build-vms vms.nix
```

Finally run `./result/bin/nixos-run-vms` and you get a VM (with passwordless root) where you can play around.

Reply:
Thanks! I'll try that when I am at home.